build.py

```diff
@@ -139,6 +142,11 @@
         exit(0)
 
     build(options.clean)
+
+    if options.flash:
+        os.chdir(base_dir)
+        flash("./MICROBIT.hex")
```

Comment:
If you have the based_dir path you can just build the full path to the hex, no need to change the current working directory, no?

Reply:
Yep - squished into 1 line 5574c45